packages/table/test/formatsTests.tsx

```diff
@@ -69,7 +71,7 @@ describe("Formats", () => {
                 majority have never stirred?
             `;
 
-            const style = { height: "200px" };
+            const style = { height: "200px", position: "relative" };
```

Comment:
NOPE. That would be bad.

![2017-08-14 11 53 27](https://user-images.githubusercontent.com/443450/29286547-45960352-80e7-11e7-8618-8ed738e1c00e.gif)

@themadcreator would you prefer I find the exact reason, or just leave the fix here, merge this PR, and revisit? I'm leaning toward the former, but if I add `position: relative`, quite a few downstream CSS fixes are required too. Might be out of scope for now.